### PR TITLE
Added my data

### PR DESCRIPTION
**Pull request by Sajjal**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [x] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [x] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by darkRaspberry** [comment]
No You don't need to edit that extra file.
You have to edit you own checklist. You filled while created this Pull request.
In _data folder u only require 2 files.

**Comment by darkRaspberry** [comment]
The checklist u filled at time of creating Pull request is only need to updated which is done.
you don't need to commit the files.
U only need 2 files in _data folder
Delete that extra files in your PR

**Comment by Sajjal** [comment]
I think it is good now. Can you please have a look?

**Comment by darkRaspberry** [comment]
still Your PR contain 11 files .
just delete extra files and leave 2 files there.
1 is your <username>.md and picture file

**Comment by Sajjal** [comment]
Removed

**Comment by darkRaspberry** [comment]
You have change 15 files.
**Your PR should only add 2 files in _data**
OK.
You have 2 options 
 - Hard way.

Restore all changes and make sure that your PR only create 2 files in _data leave other files untouched.

 - Easy way

Create a new PR.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-- [ ] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
-- [ ] My application belongs to:
-  - [ ] **Tier 1** 📖: I added myself to the Year Book.
-  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
-  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
-- [ ] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
-- [ ] I understand I should not tag or add a reviewer to this Pull Request.
-- [ ] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
-- [ ] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.
-
```
Comment (darkRaspberry) [inline .github/PULL_REQUEST_TEMPLATE.md:19]: Restore this file with original

File: .gitignore [hunk 2]
```diff
@@ -1,9 +0,0 @@
-# OS generated files #
-######################
-.DS_Store
-.DS_Store?
-._*
-.Spotlight-V100
-.Trashes
-ehthumbs.db
-Thumbs.db
```
Comment (darkRaspberry) [inline .gitignore:9]: Restore

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,22 +0,0 @@
...
-
-Attendees violating these rules will be asked to leave the event at the sole discretion of the event managers.
-
-Thank you for helping to make this a welcoming, friendly space for all.
-
-## Contact Information 📧
-If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact [support](https://support.github.com/) or another GitHub employee.
-
-
-GitHubbers will be happy to help participants contact security or local law enforcement, escort to safety, or otherwise assist those experiencing harassment to feel safe for the duration of the conference. We value your attendance.
```
Comment (darkRaspberry) [inline CODE_OF_CONDUCT.md:22]: Restore

File: README.md [hunk 4]
```diff
@@ -1,88 +0,0 @@
...
-
-Go through the checklist on the pull request template to guarantee your submission is valid. The GitHub Education team will review your application, approve and merge your submission if everything is correct. Otherwise, you will get notified of the changes requested on the pull request comment section. 
-
-# Graduation Day 🎓
-Don't forget to watch the livestream! 
-
-- 📆 Monday, June 15th 2020
-- ⏰ 9:00 PDT | 21:30 IST
-- 📍 GitHub Education Twitch Channel
-- 📎 Add the [event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to your calendar.
```
Comment (darkRaspberry) [inline README.md:88]: Restore

File: _data/CDDelta/CDDelta.md [hunk 5]
```diff
@@ -1,7 +0,0 @@
----
-name: JY
-institution: Mount Waverley Secondary College
-profile_pic: cddelta.png
-quote: In order to understand recursion, one must first understand recursion, in order to understand...
-github_user: cddelta
----
```
Comment (darkRaspberry) [inline _data/CDDelta/CDDelta.md:7]: Restore

File: _data/MonaTheOctocat/MonaTheOctocat.md [hunk 6]
```diff
@@ -1,7 +0,0 @@
----
-name: Mona Lisa # No longer than 18 characters
-institution: Octocat University # no longer than 58 characters
-profile_pic: mona.png # Relative path to the image you uploaded
-quote: "Thanks to teachers and fellow students who encouraged me to get here. I'm going to miss you all!" # No longer than 100 characters
-github_user: MonaTheOctocat
----
```
Comment (darkRaspberry) [inline _data/MonaTheOctocat/MonaTheOctocat.md:7]: Restore

File: _data/m1guelpf/m1guelpf.md [hunk 7]
```diff
@@ -1,7 +0,0 @@
----
-name: Miguel Piedrafita
-institution: Colegio San Ignacio
-profile_pic: m1guelpf.png
-quote: "Coding is the superpower anyone can get"
-github_user: m1guelpf
----
```
Comment (darkRaspberry) [inline _data/m1guelpf/m1guelpf.md:7]: Restore

File: _data/vihanb/vihanb.md [hunk 8]
```diff
@@ -1,7 +0,0 @@
----
-name: Vihan Bhargava
-institution: California High School
-profile_pic: vihanb.png
-quote: Maybe life ain't too heck.
-github_user: vihanb
----
```
Comment (darkRaspberry) [inline _data/vihanb/vihanb.md:7]: Restore

**Comment by Sajjal** [comment]
Restored and made suggested changes.

**Comment by darkRaspberry** [comment]
Yup. you made it.
Everything is correct wait till merged.

**Comment by Sajjal** [comment]
Is it? Great! Thank you for your help and support.

**Comment by darkRaspberry** [comment]
Now everything is fine.
Except this.
**Accepted files are png and jpg, should be squared and minimum size 544x544 pixels**

**Comment by Sajjal** [comment]
I have updated the image.

**Comment by darkRaspberry** [comment]
@Sajjal that's really nice.
Now just will it will be merged.

**Comment by Sajjal** [comment]
Sure. Thank you.

**Comment by Sajjal** [comment]
@darkRaspberry How long will it take to merge? If you don't mind.

**Comment by Sajjal** [comment]
Hi @juanpflores I first filled the shipping form on May 20th. I probably used "sajjal" as my github username instead of "Sajjal"

I filled the shipping form again few minutes ago. Can you please have a look?

**Comment by juanpflores** [comment]
I'll check again

**Pull request merged**